Documentation/v2/admin_guide.md

```diff
@@ -45,12 +45,12 @@ It is important to monitor your production etcd cluster for healthy information
 
 #### Health Monitoring
 
-At lowest level, etcd exposes health information via HTTP at `/health` in JSON format. If it returns `{"health": "true"}`, then the cluster is healthy. Please note the `/health` endpoint is still an experimental one as in etcd 2.2.
+At lowest level, etcd exposes health information via HTTP at `/health` in JSON format. If it returns `{"health":true}`, then the cluster is healthy. Please note the `/health` endpoint is still an experimental one as in etcd 2.2.
```

Comment:
Is this really still experimental? It seems people depend on cluster-health which depends on /health